inc/thorvg.h

```diff
@@ -1236,7 +1235,7 @@ class TVG_EXPORT SwCanvas final : public Canvas
      *
      * @warning It's not allowed after pushing any paints.
      *
-     * @BETA_API
```

Comment:
`    * @param[in] policy Use the shared cache memory. The default value is @c true`
It's confusing, since 'policy' is an enum, not a bool

` * @retval Result::InsufficientCondition If the canvas has any paints.`
any -> no

Reply:
@mgrudzinska good catch, could you please update the doc with a separate patch?  @c true is a typo, it can be changed to the default enum value.